tests/Microsoft.Bot.Builder.AI.LUIS.Tests/LuisV3OracleTests.cs

```diff
@@ -80,19 +79,20 @@ public void EmptyEndpoint()
 
             // Assert
             var app = (LuisRecognizerOptions)fieldInfo.GetValue(recognizerEmpty);
-            Assert.AreEqual("https://westus.api.cognitive.microsoft.com", app.Application.Endpoint);
+            Assert.Equal("https://westus.api.cognitive.microsoft.com", app.Application.Endpoint);
         }
 
-        [TestMethod]
-        [ExpectedException(typeof(ArgumentNullException))]
+        [Fact]
         public void LuisRecognizer_NullLuisAppArg()
         {
-            var recognizerWithNullLuisApplication = new LuisRecognizer(new LuisRecognizerOptionsV3(null));
-            Assert.Fail();
-            Assert.IsNotNull(recognizerWithNullLuisApplication);
+            Assert.Throws<ArgumentNullException>(() =>
+            {
+                var recognizerWithNullLuisApplication = new LuisRecognizer(new LuisRecognizerOptionsV3(null));
+                Assert.NotNull(recognizerWithNullLuisApplication);
```

Comment:
Is this line executed?

Reply:
It is not, do you think we should remove it? Maybe it was needed as a precaution for MSTests tests, I think if I could remember, there are more potential cases like this one.

Reply:
Yes, there're a few more cases like this, I think we should remove those unused asserts.